.github/workflows/build.yml

```diff
@@ -27,8 +27,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        image_name: [silverblue, kinoite, vauxite]
-        major_version: [37]
+        image_name: [silverblue, kinoite, vauxite, sericea, base]
+        major_version: [37, 38]
```

Comment:
I haven't gotten F38 to build, so you will have to defer F38 support to another pull request.

Reply:
Yeah F38 appears to have a packages not found error... Didn't notice that before PR.
```
error: Packages not found: xorg-x11-drv-nvidia-*:525.*.fc38.x86_64, xorg-x11-drv-nvidia-cuda*:525.*.fc38.x86_64, xorg-x11-drv-nvidia-devel*:525.*.fc38.x86_64, xorg-x11-drv-nvidia-kmodsrc*:525.*.fc38.x86_64, xorg-x11-drv-nvidia-power*:525.*.fc38.x86_64
```
I assume the F37 errors were before you added that on-the-fly keygen for PRs. I'll try to look into those errors.

Reply:
In the meantime, F38 can just be removed from the major version matrix, as the exclude should exclude sericea from trying to build on F37.

Reply:
Yeah, I think you should have an easier time getting the CI to pass if you rebase against the latest changes in `main`.

Reply:
Still appears to be having akmods errors

Reply:
It looks like the image can technically build, but there's a permission error from doing the push to the registry. I think I'll have to add another change to skip pushing for PRs.